src/figuresClasses.ts

```diff
@@ -1,19 +1,69 @@
-export interface Figure {
+type ShapesType = 'triangle' | 'circle' | 'rectangle';
+type ColorsType = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  shape : ShapesType,
+  color: ColorsType,
+  getArea(): number,
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape : ShapesType = 'triangle';
 
-}
+  constructor(
+    public color: ColorsType,
+    public a: number = 0,
+    public b: number = 0,
+    public c: number = 0,
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0 || a + b <= c) {
+      throw new Error('Enter valid values');
+    }
+  }
 
-export class Circle {
+  getArea(): number {
+    const s = (this.a + this.b + this.c) / 2;
+    const area = Math.sqrt(s * (s - this.a) * (s - this.b) * (s - this.c));
 
+    return Math.floor(area * 100) / 100;
```

Comment:
We using same rounding in all cases, so it will be nice to create helper function for it (not critical, but as for me - more logical)